src/dbus/types.rs

```diff
@@ -167,9 +168,23 @@ impl SharedMprisState {
             metadata: None,
             has_prev: false,
             has_next: false,
+            volume: 1f64,
         })))
     }
 
+    pub fn volume(&self) -> f64 {
+        self.0
+            .lock()
+            .map(|s| s.volume)
+            .map_err(|e| error!("Failed to get volume: {:?}", e))
+            .unwrap_or(1.0f64)
+    }
+    pub fn set_volume(&self, volume: f64) {
```

Comment:
(nitpick) could use a space above the function

Reply:
noted